components/motor/motor_test.go

```diff
@@ -153,8 +153,8 @@ func TestCreateStatus(t *testing.T) {
 	status := &pb.Status{IsPowered: true, PositionReporting: true, Position: 7.7, IsMoving: true}
 
 	injectMotor := &inject.LocalMotor{}
-	injectMotor.IsPoweredFunc = func(ctx context.Context, extra map[string]interface{}) (bool, error) {
-		return status.IsPowered, nil
+	injectMotor.IsPoweredFunc = func(ctx context.Context, extra map[string]interface{}) (bool, float64, error) {
+		return status.IsPowered, 2.0 /* TODO: update this if it gets used */, nil
```

Comment:
why 2.0?

Reply:
Turns out this value is completely unused. I wanted to make it `nil` but Go didn't like that for a float. So, I made it a nonsensical value, in the hopes that someone changes it to something sane if it ever becomes used (and fails some test if that doesn't happen). 

...in hindsight, setting it to 1.0 (i.e., full power) is probably reasonable for any test that uses it in the future. I'll go make that change.